src/app/clusters/service-area-server/service-area-cluster-objects.h

```diff
@@ -38,8 +38,17 @@ struct LocationStructureWrapper : public chip::app::Clusters::ServiceArea::Struc
 {
     LocationStructureWrapper()
     {
-        Set(0, 0, CharSpan(), DataModel::Nullable<int16_t>(), DataModel::Nullable<AreaTypeTag>(),
-            DataModel::Nullable<LandmarkTag>(), DataModel::Nullable<PositionTag>(), DataModel::Nullable<FloorSurfaceTag>());
+        SetLocationID(0);
+        SetMapID(DataModel::Nullable<uint8_t>());
+        SetLandmarkTag(DataModel::Nullable<LandmarkTag>());
```

Comment:
Note from @bzbarsky-apple is a separate PR: "While you are here, just using NullNullable here would be shorter, but maybe we want to make it clear which things we are leaving null?"

Reply:
Please use `NullNullable` everywhere a default null is expected.

Reply:
updated